docs/getting-started/new-project-guide/go_lang.md

```diff
@@ -27,6 +27,14 @@ only. In that mode, fuzz targets for Go use the libFuzzer engine with native Go
 coverage instrumentation. Binaries compiled in this mode provide the same
 libFuzzer command line interface as non-Go fuzz targets.
 
+## Go version
+
+Go 1.16 and below is not supported. Projects with an unsupported Go version in their `go.mod` file will need to install a newer version in their OSS-Fuzz build scripts, for example by adding this line to their `Dockerfile`:
```

Comment:
I don't know anything about go but it's surprising to me that this is necessary.

Reply:
Also a little confusing to me. Why can't users just use the Go that we provide in our base image, which is up to date?